src/api/restaurantVenues/restaurantVenues.controller.ts

```diff
@@ -85,3 +86,20 @@ export const deleteRestaurantVenuesController = async (
     next(error);
   }
 };
+
+export const updateVenueImageController = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const { id } = req.params;
+    const venue_photo = req.body.venue_photo;
+    const restaurantVenues = await updateVenueImage(id, venue_photo);
+    res
+      .status(200)
+      .json({ message: "Venue image updated", data: restaurantVenues });
```

Comment:
is necessary not returning the response?

Reply:
Resolved